src/node_crypto.cc

```diff
@@ -6623,6 +6626,65 @@ class NidKeyPairGenerationConfig : public KeyPairGenerationConfig {
   const int id_;
 };
 
+struct PrimeInfo {
+  BignumPointer fixed_value_;
```

Comment:
The prime can be any fixed value, not just primes that are part of a known group. I am open to other names, but I think `modp_group_prime_` is too specific.

Reply:
I think the name is fine, but I do think its worth some comments on the relationship between the two. I'm guessing that `prime_size_` can be set and the fixed_value_ be null? Because otherwise it wouldn't have to exist, bignums know their size. What about the other way?